composer/spec/fixtures/projects/v1/invalid_v2_name/composer.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "this name is no longer valid in composerv2",
```

Comment:
woah this was valid in v1 🙈

Reply:
Yeah, it's been throwing warnings about these for a while though 😄 but we run into it reasonably often still, I'm guessing on older packages mostly